### PR TITLE
Require go version 1.16 and check go version

### PR DESCRIPTION
**Pull request by jfrohnhofen**

### CODE REVIEW

**Comment by ljanyst** [comment]
Go 2 is coming. Would that check be easily adjustable?

**Comment by jfrohnhofen** [comment]
Updated to also check the major version. Also added commands to install go 1.16 to the readme.

File: cmd/build.go [hunk 1]
```diff
@@ -435,7 +458,7 @@ func createModFileContent(moduleName string, modules map[string]string, pathPref
 	mod := strings.Builder{}
 
 	fmt.Fprintf(&mod, "module %s\n\n", moduleName)
-	fmt.Fprintf(&mod, "go %s\n\n", goVersion)
+	fmt.Fprintf(&mod, "go 1.%d\n\n", goMinorVersion)
```
Comment (ljanyst) [inline cmd/build.go:461]: This still ignores the major.

**Review by ljanyst — APPROVED** [review]
LGTM module the last comment.

File: cmd/build.go (hunk 1 above)
Comment (jfrohnhofen) [reply cmd/build.go]: Done

**Pull request merged**